src/scss/_fullscreen.scss

```diff
@@ -0,0 +1,38 @@
+////
+/// @group o-overlay
+/// @link http://registry.origami.ft.com/components/o-overlay
+////
+
+///
+/// Add to the overlay for a for it to stretch to full width or full height
+///
+/// @param {String} fill - 'width' or 'height'
+///
+@mixin oOverlayFullscreen($fill) {
+	overflow: hidden;
+
+	@if ($fill == 'width') {
+		padding-left: 0;
+		padding-right: 0;
+		width: 100%;
+		left: 0;
+	} @else if ($fill == 'height') {
+		padding-top: 0;
+		padding-bottom: 0;
+		height: 100%;
+		top: 0;
+	}
+}
+
+///
+/// Add to the content for a for it to stretch to full width or full height
+///
+/// @param {String} fill - 'width' or 'height'
+///
+@mixin oOverlayFullscreenContent($fill) {
```

Comment:
Writing a mixin to output either `width: 100%` or `height: 100%` seems overkill.

If you think it absolutely makes sense to have it, could be written in one line:

```scss
#{$fill}: 100%;
```